forge/ee/routes/flowBlueprints/index.js

```diff
@@ -269,6 +269,45 @@ module.exports = async function (app) {
         })
     })
 
+    if (app.config.blueprintImport?.export) {
+        app.get('/export-public', {
+            config: {
+                allowAnonymous: true
+            },
+            schema: {
+                summary: 'Export one or more Blueprints',
+                tags: ['Flow Blueprints'],
+                query: {
+                    id: { type: 'array', items: { type: 'string' } }
+                },
+                response: {
+                    200: {
+                        type: 'object',
+                        allOf: [{ $ref: 'FlowBlueprintExport' }],
+                        properties: {
+                            count: { type: 'integer' }
+                        }
+                    },
+                    '4xx': {
+                        $ref: 'APIError'
+                    }
+                }
+            }
+        }, async (request, reply) => {
+            let where = {}
```

Comment:
```suggestion
            let where = { active: true }
```

We should only include active blueprints - not expose any work-in-progress or ones that we've chosen to hide.